snap/snapcraft.yaml

```diff
@@ -65,15 +74,14 @@ parts:
     build-attributes:
       - enable-patchelf
     prime:
-      - "usr/lib/git-core"
-      - "usr/share/git-core"
-      - "usr/lib/x86_64-linux-gnu/libcurl-gnutls*"
-      - "usr/lib/x86_64-linux-gnu/libpsl*"
-      - "usr/lib/x86_64-linux-gnu/librtmp*"
-      - "usr/lib/x86_64-linux-gnu/libnghttp2*"
-      - "usr/lib/x86_64-linux-gnu/libldap*"
-      - "usr/lib/x86_64-linux-gnu/liblber*"
-      - "usr/lib/x86_64-linux-gnu/libsasl2*"
+      - "-usr/bin"
+      - "-usr/share/doc"
+      - "-usr/share/man"
+      # perl is part of the core22 / core24
```

Comment:
are we using perl itself from the core24 base? If so, that could be a problem as it is not `classic` ready